microsoft-edge/devtools-guide-chromium/accessibility/lighthouse.md

```diff
@@ -0,0 +1,91 @@
+---
+description: Testing accessibility using Lighthouse from within DevTools.
+title: Testing accessibility using Lighthouse
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.date: 05/13/2021
+ms.topic: article
+ms.prod: microsoft-edge
+keywords: microsoft edge, web development, f12 tools, devtools
+---
+<!-- this article was created on 05/11/2021 by moving a section out from the "Accessibility reference" article (reference.md) -->
+<!-- Copyright Kayce Basques 
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       https://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.  -->  
+
+# Testing accessibility using Lighthouse
+
+You can use Lighthouse from within DevTools to audit the accessibility of a page and generate a report, as follows.
+
+> [!NOTE]
+> The **Lighthouse** tool provides links to content hosted on third-party websites.  Microsoft is not responsible for and has no control over the content of these sites and any data that may be collected.  
+
+Use the Lighthouse tool to determine:
+
+*   Whether a page is properly marked up for screen readers.  
+*   Whether the text elements on a page have sufficient contrast ratios, as shown in the [Color Picker](color-picker.md).  
+
+To audit a page:
+
+1.  Navigate to the URL that you want to audit.
+1.  In DevTools, select the **Lighthouse** tool (previously labelled as the **Audits** tool).  DevTools shows you various configuration options.
+    
+    :::image type="complex" source="../media/accessibility-audits-pane.msft.png" alt-text="Configure audits" lightbox="../media/accessibility-audits-pane.msft.png":::
+       Configure audits  
+    :::image-end:::  
```

Comment:
```suggestion
    :::image type="complex" source="../media/accessibility-audits-pane.msft.png" alt-text="Configuration options of the Lighthouse tool" lightbox="../media/accessibility-audits-pane.msft.png":::
       Configuration options of the Lighthouse tool  
    :::image-end:::  
```

Reply:
Can we update the image since this one shows Audit but the product shows Lighthouse?